### PR TITLE
refactor: update to a more skyra-like build stack

### PR DESCRIPTION
**Pull request by favna**
> **Warning**
> Payment Required

### CODE REVIEW

File: .github/workflows/continuous-delivery.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          submodules: true
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6 # tag=v2.0.0
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@49ed152c8eca782a232dede0303416e8f356c37b # tag=v2.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Skyra Docker image
```
Comment (kyranet) [inline .github/workflows/continuous-delivery.yml:31]:
```suggestion
      - name: Build and push Alestra Docker image
```

File: .github/renovate.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["github>sapphiredev/readme:sapphire-renovate"],
+	"labels": ["Meta: Dependencies"],
+	"npm": {
+		"packageRules": [
+			{
+				"matchPackagePatterns": [
+					"@napi-rs/canvas",
```
Comment (kyranet) [inline .github/renovate.json:9]: Why is this ignored?

**Review by kyranet — CHANGES_REQUESTED** [review]

File: .github/renovate.json (hunk 2 above)
Comment (favna) [reply .github/renovate.json:9]: For the same reason it is ignored in skyra and canvas-constructor. Code doesn't compile with the latest version. Informed you at the time but you're too busy to fix it 💀

File: .github/renovate.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["github>sapphiredev/readme:sapphire-renovate"],
+	"labels": ["Meta: Dependencies"],
```
Comment (kyranet) [inline .github/renovate.json:4]: Won't labelsync cause this label to be outdated?

**Review by kyranet — CHANGES_REQUESTED** [review]

File: .github/renovate.json (hunk 3 above)
Comment (favna) [reply .github/renovate.json]: Fixed

**Review by kyranet — APPROVED** [review]

**Pull request merged**